jubakit/loader/postgresql.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ..base import BaseLoader
+from ..compat import *
+from psycopg2 import connect
+from psycopg2.extras import DictCursor
+from psycopg2 import sql
+
+class PostgreSQLoader(BaseLoader):
+  """
+  Loader to process columns of PostgreSQL.
+  """
+
+  def __init__(self, auth, **kwargs):
+    self.auth = auth
+    if 'table' not in kwargs:
+      raise RuntimeError('table name is not specify.')
+    self.kwargs = kwargs
+
+  def rows(self):
+    with connect(self.auth.get()) as connection:
+      with connection.cursor(cursor_factory=DictCursor) as cursor:
+        cursor.execute(
+          sql.SQL("SELECT * FROM {}")
+          .format(sql.Identifier(self.kwargs['table'])))
+        return cursor.fetchall()
```

Comment:
In [jubakit loader implementation guide](http://jubat.us/ja/jubakit/guide/loader_develop.html#implementing-new-loader), rows function must use `yield`, not `return`. So would you fix it?

In addition, the return values in rows function should be the `dict` type. The key should be the `column` name. 
For example, the [array loader](https://github.com/jubatus/jubakit/blob/master/jubakit/loader/array.py#L23-L32) set feature_names in the constructor and use it to make the dict object. 
So would you fix it?